recipes/mmtf-pyspark/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: mmtfpyspark
+  version: "0.2.0"
+
+source:
```

Comment:
It looks like this package is available from pypy.io, so please install from there:

https://pypi.org/project/mmtfPyspark/
https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos